_posts/2021-02-10-webhooks.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Webhooks in sensenet"
+author: [tusmester]
+image: "../img/posts/webhooks/header.png"
+tags: [webhooks, API, saas, event, service]
+redirect_to: https://www.sensenet.com/blog/2021-02-10-webhooks
+---
+
+Webhooks have (finally) arrived to sensenet!
+
+---
+
+In modern service platforms like sensenet developers need the possibility to do something when something happens in the service. In more technical terms: handling **events** is one of the most important things we need when building an enterprise application. 
+
+In the old days backend developers would handle this issue by developing a module that runs inside the application when a user uploads a file - but since everything has moved to the cloud, this is not an option. 
```

Comment:
```suggestion
In the old days backend developers would handle this issue by developing a module that runs inside the application when a user uploads a file - but since everything has moved to the cloud, this is not an option anymore. 
```